src/Dialog/confirmation-dialog.scss

```diff
@@ -0,0 +1,10 @@
+@import "../variables";
+
+.modal {
+    background-color: $modal-backdrop-bg;
+}
+
+.modal-content {
+    border-radius: 0px !important;
+    border-color: $modal-content-border-color;
```

Comment:
My impression is, that this gets overridden anyhow, so you can remove it.

Reply:
Removed.